docs/reference/ccr/index.asciidoc

```diff
@@ -118,11 +118,17 @@ comes online, replication resumes between the clusters.
 
 image::images/ccr-arch-bi-directional.png[Bi-directional configuration where each cluster contains both a leader index and follower indices]
 
-NOTE: This configuration is useful for index-only workloads, where no updates
+This configuration is useful for index-only workloads, where no updates
 to document values occur. In this configuration, documents indexed by {es} are
 immutable. Clients are located in each datacenter alongside the {es}
 cluster, and do not communicate with clusters in different datacenters.
 
+NOTE: Follower indices will reject writes in a bi-directional configuration.
```

Comment:
Follower indices always reject writes, but this wording suggests there's something special about bi-directional replication in this regard. You always have to go to the leader index to make changes. Can we make this information more generally applicable?

Reply:
@DaveCTurner, after some review, I decided to pull that information out of a note, create a new sub-section for processing updates, and merge the information there. Here's a preview link: https://elasticsearch_72266.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/xpack-ccr.html#ccr-update-leader-index